index.html

```diff
@@ -137,11 +163,27 @@ <h4>background-image</h4>
           </li>
           <li>
             <h4>controls</h4>
-            <P>Enables controls via mouse/touch when in flat view.</P>
+            <p>Enables controls via mouse/touch when in flat view.</p>
           </li>
           <li>
-            <h4>controls</h4>
-            <P>Enables controls via mouse/touch when in flat view.</P>
+            <h4>environment-image</h4>
+            <p>Controls the environmental reflection of the model. Normally if
+            background-image is set, that image will also be used for the
+            environment-image. Use environment-image to only set the reflection
+            without affecting the background.</p>
+          </li>
+          <li>
+            <h4>environment-intensity</h4>
+            <p>Controls the intensity of the environmental reflection. There is
+            always a reflection from the environment, even when background-image
+            or environment-image are not set. Set this value to 0 to hide the
+            environment entirely. Defaults to 1.</p>
+          </li>
+          <li>
+            <h4>experimental-pmrem</h4>
+            <p>Enables an experimental lighting mode that is more physically
+            realistic, and emphasizes light from the environment over light from
+            the built-in scene lights.</p>
```

Comment:
Would be good to add a link here for those interested in the details; [here](https://wiki.jmonkeyengine.org/jme3/advanced/pbr_part3.html) is a link describing what's going on (not the most ideal link considering a skill diversity of audience, but sufficient)

Reply:
👍 Will fix